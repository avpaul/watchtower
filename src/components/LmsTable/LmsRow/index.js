import LmsRow from './LmsRow';

export default LmsRow;
