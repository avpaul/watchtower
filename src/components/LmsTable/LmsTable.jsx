import React, { Fragment } from 'react';
import PropTypes from 'prop-types';
import arrayKey from 'weak-key';
import TableHeader from '../TableComponents/Header';
import Table from '../TableComponents/Table';
import Error from '../Error';
import Loader from '../Loader/Loader';
import LmsRow from './LmsRow';

const LmsTable = ({ lmsSubmissions, loading, fellow }) => {
  const { ErrorMessage } = Error;

  if (!lmsSubmissions) {
    return (
      <ErrorMessage message="There's currently no LMS data for this fellow" />
    );
  }
  const headers = [
    'Course Id',
    'Output Name',
    'Due Date',
    'Submission Date',
    'Review Date',
    'Score'
  ];
  return (
    <Fragment>
      <div className="lms-table">
        <Table>
          <TableHeader headers={headers} />
          {!!lmsSubmissions &&
            lmsSubmissions.map(lmsSubmission =>
              lmsSubmission.level === fellow.level ? (
                <LmsRow
                  key={arrayKey(lmsSubmission)}
                  lmsSubmission={lmsSubmission}
                />
              ) : (
                ''
              )
            )}
        </Table>
      </div>
      {loading && <Loader />}
    </Fragment>
  );
};

LmsTable.propTypes = {
  lmsSubmissions: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
  loading: PropTypes.bool.isRequired,
  fellow: PropTypes.shape({}).isRequired
};

export default LmsTable;
