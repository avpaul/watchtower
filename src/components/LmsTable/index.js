import LmsTable from './LmsTable';

export default LmsTable;
