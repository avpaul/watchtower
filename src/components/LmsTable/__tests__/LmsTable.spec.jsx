import React from 'react';
import { shallow } from 'enzyme';
import LmsTable from '../LmsTable';

describe('tests the LmsTable', () => {
  let wrapper;
  let wrapper2;
  let wrapper3;
  beforeEach(() => {
    const props = {
      lmsSubmissions: [],
      loading: true,
      fellow: {}
    };
    const props2 = {
      lmsSubmissions: [],
      loading: false,
      fellow: {
        actualApprenticeshipStartDate: '2019-03-18',
        actualSimulationsCompletionDate: '2019-03-15',
        apprenticeshipManager: 'Trust Birungi',
        apprenticeshipTeam: 'Watch Tower',
        apprenticeshipTechnology:
          'JavaScript,  PHP/Laravel,  ReactJS,  Eloquent',
        bio:
          'Sed quo voluptatum ducimus sunt labore eos totam. Sit corporis est voluptates commodi occaecati. Modi harum assumenda quod voluptatem.',
        cohort: 'Class 13 - KLA',
        currentStage: 'Apprentice',
        email: 'stanton.rogahn@andela.com'
      }
    };

    const props3 = {
      lmsSubmissions: [{}],
      loading: true,
      fellow: {
        actualApprenticeshipStartDate: '2019-03-18',
        actualSimulationsCompletionDate: '2019-03-15',
        apprenticeshipManager: 'Trust Birungi',
        apprenticeshipTeam: 'Watch Tower',
        apprenticeshipTechnology:
          'JavaScript,  PHP/Laravel,  ReactJS,  Eloquent',
        bio:
          'Sed quo voluptatum ducimus sunt labore eos totam. Sit corporis est voluptates commodi occaecati. Modi harum assumenda quod voluptatem.',
        cohort: 'Class 13 - KLA',
        currentStage: 'Apprentice',
        email: 'stanton.rogahn@andela.com'
      }
    };

    wrapper = shallow(<LmsTable {...props} />);
    wrapper2 = shallow(<LmsTable {...props2} />);
    wrapper3 = shallow(<LmsTable {...props3} />);
  });
  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
  it('renders correctly', () => {
    expect(wrapper2).toMatchSnapshot();
  });
  it('renders correctly', () => {
    expect(wrapper3).toMatchSnapshot();
  });
});
