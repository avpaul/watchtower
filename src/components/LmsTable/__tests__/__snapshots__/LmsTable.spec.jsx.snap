// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests the LmsTable renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <LmsTable
    fellow={Object {}}
    lmsSubmissions={Array []}
    loading={true}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <div
          className="lms-table"
>
          <Table>
                    <TableHeader
                              headers={
                                        Array [
                                                  "Course Id",
                                                  "Output Name",
                                                  "Due Date",
                                                  "Submission Date",
                                                  "Review Date",
                                                  "Resubmitted",
                                                  "Score",
                                                ]
                              }
                    />
          </Table>
</div>,
        <Loader
          size=""
/>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Table>
            <TableHeader
                        headers={
                                    Array [
                                                "Course Id",
                                                "Output Name",
                                                "Due Date",
                                                "Submission Date",
                                                "Review Date",
                                                "Resubmitted",
                                                "Score",
                                              ]
                        }
            />
</Table>,
          "className": "lms-table",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <TableHeader
                headers={
                                Array [
                                                "Course Id",
                                                "Output Name",
                                                "Due Date",
                                                "Submission Date",
                                                "Review Date",
                                                "Resubmitted",
                                                "Score",
                                              ]
                }
/>,
              Array [],
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "headers": Array [
                  "Course Id",
                  "Output Name",
                  "Due Date",
                  "Submission Date",
                  "Review Date",
                  "Resubmitted",
                  "Score",
                ],
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "size": "",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <div
            className="lms-table"
>
            <Table>
                        <TableHeader
                                    headers={
                                                Array [
                                                            "Course Id",
                                                            "Output Name",
                                                            "Due Date",
                                                            "Submission Date",
                                                            "Review Date",
                                                            "Resubmitted",
                                                            "Score",
                                                          ]
                                    }
                        />
            </Table>
</div>,
          <Loader
            size=""
/>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Table>
              <TableHeader
                            headers={
                                          Array [
                                                        "Course Id",
                                                        "Output Name",
                                                        "Due Date",
                                                        "Submission Date",
                                                        "Review Date",
                                                        "Resubmitted",
                                                        "Score",
                                                      ]
                            }
              />
</Table>,
            "className": "lms-table",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": Array [
                <TableHeader
                  headers={
                                    Array [
                                                      "Course Id",
                                                      "Output Name",
                                                      "Due Date",
                                                      "Submission Date",
                                                      "Review Date",
                                                      "Resubmitted",
                                                      "Score",
                                                    ]
                  }
/>,
                Array [],
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "headers": Array [
                    "Course Id",
                    "Output Name",
                    "Due Date",
                    "Submission Date",
                    "Review Date",
                    "Resubmitted",
                    "Score",
                  ],
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": [Function],
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "size": "",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;

exports[`tests the LmsTable renders correctly 2`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <LmsTable
    attempts={1}
    fellow={
        Object {
            "actualApprenticeshipStartDate": "2019-03-18",
            "actualSimulationsCompletionDate": "2019-03-15",
            "apprenticeshipManager": "Trust Birungi",
            "apprenticeshipTeam": "Watch Tower",
            "apprenticeshipTechnology": "JavaScript,  PHP/Laravel,  ReactJS,  Eloquent",
            "bio": "Sed quo voluptatum ducimus sunt labore eos totam. Sit corporis est voluptates commodi occaecati. Modi harum assumenda quod voluptatem.",
            "cohort": "Class 13 - KLA",
            "currentStage": "Apprentice",
            "email": "stanton.rogahn@andela.com",
          }
    }
    lmsSubmissions={Array []}
    loading={false}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <div
          className="lms-table"
>
          <Table>
                    <TableHeader
                              headers={
                                        Array [
                                                  "Course Id",
                                                  "Output Name",
                                                  "Due Date",
                                                  "Submission Date",
                                                  "Review Date",
                                                  "Resubmitted",
                                                  "Score",
                                                ]
                              }
                    />
          </Table>
</div>,
        false,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Table>
            <TableHeader
                        headers={
                                    Array [
                                                "Course Id",
                                                "Output Name",
                                                "Due Date",
                                                "Submission Date",
                                                "Review Date",
                                                "Resubmitted",
                                                "Score",
                                              ]
                        }
            />
</Table>,
          "className": "lms-table",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <TableHeader
                headers={
                                Array [
                                                "Course Id",
                                                "Output Name",
                                                "Due Date",
                                                "Submission Date",
                                                "Review Date",
                                                "Resubmitted",
                                                "Score",
                                              ]
                }
/>,
              Array [],
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "headers": Array [
                  "Course Id",
                  "Output Name",
                  "Due Date",
                  "Submission Date",
                  "Review Date",
                  "Resubmitted",
                  "Score",
                ],
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        "type": "div",
      },
      false,
    ],
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <div
            className="lms-table"
>
            <Table>
                        <TableHeader
                                    headers={
                                                Array [
                                                            "Course Id",
                                                            "Output Name",
                                                            "Due Date",
                                                            "Submission Date",
                                                            "Review Date",
                                                            "Resubmitted",
                                                            "Score",
                                                          ]
                                    }
                        />
            </Table>
</div>,
          false,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Table>
              <TableHeader
                            headers={
                                          Array [
                                                        "Course Id",
                                                        "Output Name",
                                                        "Due Date",
                                                        "Submission Date",
                                                        "Review Date",
                                                        "Resubmitted",
                                                        "Score",
                                                      ]
                            }
              />
</Table>,
            "className": "lms-table",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": Array [
                <TableHeader
                  headers={
                                    Array [
                                                      "Course Id",
                                                      "Output Name",
                                                      "Due Date",
                                                      "Submission Date",
                                                      "Review Date",
                                                      "Resubmitted",
                                                      "Score",
                                                    ]
                  }
/>,
                Array [],
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "headers": Array [
                    "Course Id",
                    "Output Name",
                    "Due Date",
                    "Submission Date",
                    "Review Date",
                    "Resubmitted",
                    "Score",
                  ],
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": [Function],
          },
          "type": "div",
        },
        false,
      ],
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;

exports[`tests the LmsTable renders correctly 3`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <LmsTable
    fellow={
        Object {
            "actualApprenticeshipStartDate": "2019-03-18",
            "actualSimulationsCompletionDate": "2019-03-15",
            "apprenticeshipManager": "Trust Birungi",
            "apprenticeshipTeam": "Watch Tower",
            "apprenticeshipTechnology": "JavaScript,  PHP/Laravel,  ReactJS,  Eloquent",
            "bio": "Sed quo voluptatum ducimus sunt labore eos totam. Sit corporis est voluptates commodi occaecati. Modi harum assumenda quod voluptatem.",
            "cohort": "Class 13 - KLA",
            "currentStage": "Apprentice",
            "email": "stanton.rogahn@andela.com",
          }
    }
    lmsSubmissions={
        Array [
            Object {},
          ]
    }
    loading={true}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <div
          className="lms-table"
>
          <Table>
                    <TableHeader
                              headers={
                                        Array [
                                                  "Course Id",
                                                  "Output Name",
                                                  "Due Date",
                                                  "Submission Date",
                                                  "Review Date",
                                                  "Resubmitted",
                                                  "Score",
                                                ]
                              }
                    />
                    <LmsRow
                              lmsSubmission={Object {}}
                    />
          </Table>
</div>,
        <Loader
          size=""
/>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Table>
            <TableHeader
                        headers={
                                    Array [
                                                "Course Id",
                                                "Output Name",
                                                "Due Date",
                                                "Submission Date",
                                                "Review Date",
                                                "Resubmitted",
                                                "Score",
                                              ]
                        }
            />
            <LmsRow
                        lmsSubmission={Object {}}
            />
</Table>,
          "className": "lms-table",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": Array [
              <TableHeader
                headers={
                                Array [
                                                "Course Id",
                                                "Output Name",
                                                "Due Date",
                                                "Submission Date",
                                                "Review Date",
                                                "Resubmitted",
                                                "Score",
                                              ]
                }
/>,
              Array [
                <LmsRow
                  lmsSubmission={Object {}}
/>,
              ],
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "headers": Array [
                  "Course Id",
                  "Output Name",
                  "Due Date",
                  "Submission Date",
                  "Review Date",
                  "Resubmitted",
                  "Score",
                ],
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": "weak-key-2",
              "nodeType": "function",
              "props": Object {
                "lmsSubmission": Object {},
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": [Function],
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "size": "",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <div
            className="lms-table"
>
            <Table>
                        <TableHeader
                                    headers={
                                                Array [
                                                            "Course Id",
                                                            "Output Name",
                                                            "Due Date",
                                                            "Submission Date",
                                                            "Review Date",
                                                            "Resubmitted",
                                                            "Score",
                                                          ]
                                    }
                        />
                        <LmsRow
                                    lmsSubmission={Object {}}
                        />
            </Table>
</div>,
          <Loader
            size=""
/>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Table>
              <TableHeader
                            headers={
                                          Array [
                                                        "Course Id",
                                                        "Output Name",
                                                        "Due Date",
                                                        "Submission Date",
                                                        "Review Date",
                                                        "Resubmitted",
                                                        "Score",
                                                      ]
                            }
              />
              <LmsRow
                            lmsSubmission={Object {}}
              />
</Table>,
            "className": "lms-table",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": Array [
                <TableHeader
                  headers={
                                    Array [
                                                      "Course Id",
                                                      "Output Name",
                                                      "Due Date",
                                                      "Submission Date",
                                                      "Review Date",
                                                      "Resubmitted",
                                                      "Score",
                                                    ]
                  }
/>,
                Array [
                  <LmsRow
                    lmsSubmission={Object {}}
/>,
                ],
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "headers": Array [
                    "Course Id",
                    "Output Name",
                    "Due Date",
                    "Submission Date",
                    "Review Date",
                    "Resubmitted",
                    "Score",
                  ],
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": "weak-key-2",
                "nodeType": "function",
                "props": Object {
                  "lmsSubmission": Object {},
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": [Function],
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "size": "",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;

exports[`tests the LmsTable renders correctly 4`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <LmsTable
    fellow={
        Object {
            "actualApprenticeshipStartDate": "2019-03-18",
            "actualSimulationsCompletionDate": "2019-03-15",
            "apprenticeshipManager": "Trust Birungi",
            "apprenticeshipTeam": "Watch Tower",
            "apprenticeshipTechnology": "JavaScript,  PHP/Laravel,  ReactJS,  Eloquent",
            "bio": "Sed quo voluptatum ducimus sunt labore eos totam. Sit corporis est voluptates commodi occaecati. Modi harum assumenda quod voluptatem.",
            "cohort": "Class 13 - KLA",
            "currentStage": "Apprentice",
            "email": "stanton.rogahn@andela.com",
          }
    }
    lmsSubmissions={null}
    loading={true}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "customStyle": Object {},
      "message": "There's currently no LMS data for this fellow",
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "customStyle": Object {},
        "message": "There's currently no LMS data for this fellow",
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;
